main.go

```diff
@@ -93,6 +96,54 @@ type config struct {
 	GenericWebhookURLs []string `yaml:"generic_webhook_urls"`
 }
 
+// RejectionCondition contains the fields that should match a bug report for it to be rejected.
+type RejectionCondition struct {
+	Version string `yaml:"version"`
+	Label   string `yaml:"label"`
+	App     string `yaml:"app"`
```

Comment:
I note that the behaviour of `App==""` is quite different to the behaviour if `Version==""` or `Label=""`. Please document those behaviours here.

(should we instead use `*string` for Label and App, so that `nil` is different to `""` ?)

Reply:
There is no distinction between nil and "" in the filter because it doesn't make any practical difference as 0-length versions/labels are nonsensical. Adding `*string` just forces us to do more nil checks.